src/appconfig.ts

```diff
@@ -0,0 +1,170 @@
+// Set default editor mode (visual or textual)
+export const initialMode = 'visual'
+// Allow user to switch editor modes
+export const allowModeSwitch: boolean = true
+
+// =============================SET DEFAULT HTML CODE HERE============================================================
+export const initialCode = "<body>" +
+  "<div>" +
+  "<h1>" +
+  "Example Domain" +
+  "</h1>" +
+  "<p>This domain is for use in illustrative examples in documents. " +
+  "You may use this domain in literature without prior coordination or asking for permission.</p>" +
+  "<p>More <i>information</i>...</p>" +
+  "<span><?i>Hello world</?i></span>" +
+  "<p>Hello!</p>" +
+  "</div>" +
+  "</body>"
+// ===================================================================================================================
+
+// =============================SET AVAILABLE TOOLBOX CATEGORIES & BLOCKS HERE========================================
+// Each category has id (manually increment for now)
+// Name of category
+// Color of category for side bar style
+// Blocks contained within category
+   // Blocks have id (manually increment for now)
+   // Tag (actual HTML tag value)
+   // Paired (whether the tag is self-closing or paired with a closing tag
+   // Desc (block description)
+export const toolboxBlocks = {
```

Comment:
You should type the variable when it's declared (ie here) instead of when it's used (in `src/toolboxconfig.ts`)